documentation/metrics_api.markdown

```diff
@@ -0,0 +1,267 @@
+---
+layout: default
+title: "Puppet Server: Metrics API"
+canonical: "/puppetserver/latest/metrics_api.html"
+---
+
+# The Metrics API
+
+Puppet Enterprise includes two optional, enabled-by-default web APIs for
+[Java Management Extension (JMX)](https://docs.oracle.com/javase/tutorial/jmx/index.html)
+metrics, namely
+[managed beans (MBeans)](https://docs.oracle.com/javase/tutorial/jmx/mbeans/).
+
+## Metrics v2
```

Comment:
Its definitely enabled by default. The example responses are edited actual responses that I got from a running instance.

I may have conflated the content in the metrics.conf by default with some of the structure in the docs (I can't resolve the host I set up to help with this right now to confirm). I definitely agree that having the metrics.conf with the default values for the main settings in there and commented out is valuable and should be done now (I guess I just thought that was already done).